src/realm/cluster.cpp

```diff
@@ -2139,10 +2139,10 @@ Obj ClusterTree::insert(ObjKey k, const FieldValues& values)
         };
         get_owner()->for_each_public_column(insert_in_column);
 
-        if (!table->is_embedded()) {
-            if (Replication* repl = table->get_repl()) {
-                repl->create_object(table, k);
-                for (const auto& v : values) {
+        if (Replication* repl = table->get_repl()) {
```

Comment:
Could you elaborate why we no longer need to check is the table is embedded?

Reply:
That was to prevent `repl->create_object(table, k);` from being called. This is now removed.

Reply:
oh, missed that.